### ISSUE TITLE
[CI] sqlite3.OperationalError: unable to open database file

### ISSUE THREAD

**Issue by caglar10ur** [vmware/vic#4695]
I sometimes see CI to fail with following exception, don't know why...

```
==============================================================================
Output:  /drone/src/github.com/vmware/vic/output.xml
Log:     /drone/src/github.com/vmware/vic/log.html
Report:  /drone/src/github.com/vmware/vic/report.html
  adding: output.xml (deflated 96%)
...
  adding: vmkernel.log (deflated 91%)
Copying file://integration_logs_9678_2bba8be0ac9db2e692cbe637d275861b11efb273.zip [Content-Type=application/zip]...
...
Operation completed over 1 objects/42.1 MiB.                                     
Stopping rpcbind daemon....
ln: failed to create symbolic link '/run/sendsigs.omit.d/rpcbind': No such file or directory
Starting rpcbind daemon... failed!
mount.nfs: No such device
Saving integration test results to /drone-test-results/testruns-db/vic-all.db
 Database |   - Establishing database connection
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/lib/python2.7/dist-packages/dbbot/run.py", line 51, in <module>
    DbBot().run()
  File "/usr/local/lib/python2.7/dist-packages/dbbot/run.py", line 31, in __init__
    self._db = DatabaseWriter(database_path, verbose_stream)
  File "/usr/local/lib/python2.7/dist-packages/dbbot/reader/database_writer.py", line 20, in __init__
    super(DatabaseWriter, self).__init__(db_file_path, verbose_stream)
  File "/usr/local/lib/python2.7/dist-packages/dbbot/robot_database.py", line 23, in __init__
    self._connection = self._connect(db_file_path)
  File "/usr/local/lib/python2.7/dist-packages/dbbot/robot_database.py", line 28, in _connect
    return sqlite3.connect(db_file_path)
sqlite3.OperationalError: unable to open database file
----------------------------------------------
```

**Comment by anchal-agrawal** [comment]
Ping @jakedsouza - I saw this in https://ci.vmware.run/vmware/vic/9773. Robot logs for that build weren't uploaded on your log server. Same for the above build (9678).

**Comment by maplain** [comment]
same https://ci.vmware.run/vmware/vic/9794

**Comment by anchal-agrawal** [comment]
Fixed by #4770.
